Dive Computer/driver_init.c

```diff
@@ -123,18 +136,32 @@ void AON_SLEEP_TIMER0_register_isr(void)
 	*temp = (uint32_t)AON_SLEEP_TIMER0_Handler;
 }
 
-void system_init(void)
+uint8_t getSoundPin() 
+{
+	return GPIO(GPIO_PORTA,7);
+}
+
+void system_init(bool spiEnabled, bool i2cEnabled, bool soundEnabled)
```

Comment:
We have define and variable for SPI_ENABLED, I2C_ENABLED...

Reply:
Next PR I will make a config struct that will control the configuration of the board.